hubstorage/serialization.py

```diff
@@ -1,4 +1,5 @@
 from json import dumps, loads
+from msgpack import Unpacker
```

Comment:
or some condition - if msgpack-python is installed - we attempt to download as msgpack, otherwise we stick with jl